django_lightweight_queue/backends/reliable_redis.py

```diff
@@ -101,6 +101,28 @@ def processed_job(self, queue, worker_number, job):
     def length(self, queue):
         return self.client.llen(self._key(queue))
 
+    def deduplicate(self, queue):
+        main_queue_key = self._key(queue)
+
+        original_size = self.client.llen(main_queue_key)
+
+        if not original_size:
+            return 0, 0
+
+        jobs = {}
+
+        for raw_data in self.client.lrange(main_queue_key, 0, -1):
+            job_id = Job.from_json(raw_data).identity_without_created()
```

Comment:
`job_id` implies (to me at least) that it's some kind of identifier - rather than the raw data of the job minus the created, which we're using to compare jobs. Can we use a different name?